specification/riskiq/Easm/routes.tsp

```diff
@@ -0,0 +1,169 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+import "./models.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Easm;
+
+alias ServiceTraits = NoRepeatableRequests &
+  NoConditionalRequests &
+  NoClientRequestId;
+
+alias Operations = Azure.Core.ResourceOperations<ServiceTraits>;
+@tag("Assets")
+@doc("Provides access to inventory assets.")
+interface Assets {
+  @doc("Retrieve a list of assets for the provided search parameters.")
+  listAssetResource is Operations.ResourceList<AssetResource, AssetsListTrait>;
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("Update labels on assets matching the provided filter.")
+  @route("/assets")
+  @post
+  updateAssets is Foundations.Operation<
+    AssetsUpdateParameter & AssetUpdateData,
+    Task
+  >;
+  @doc("Retrieve an asset by assetId.")
+  getAssetResource is Operations.ResourceRead<AssetResource>;
+}
+
+@tag("Data Connections")
+@doc("Provides access to data connection resources.")
+interface DataConnections {
+  @doc("Retrieve a list of data connections.")
+  listDataConnection is Operations.ResourceList<
+    DataConnection,
+    DataConnectionsListTrait
+  >;
+  @doc("Validate a data connection with a given dataConnectionName.")
+  @action("validate")
+  validateDataConnection is Operations.ResourceCollectionAction<
+    DataConnection,
+    DataConnectionData,
+    ValidateResult
+  >;
+  @doc("Retrieve a data connection with a given dataConnectionName.")
+  getDataConnection is Operations.ResourceRead<DataConnection>;
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("Create or replace a data connection with a given dataConnectionName.")
+  @put
```

Comment:
This should add a decorator to indicate that is an update or createOrUpdate  or createOrReplace Operation for DataConnection,

likely `@TypeSpec.Rest.createsOrReplacesResource(DataConnection)`